pyca/config.py

```diff
@@ -165,6 +169,8 @@ def logger_init():
         handlers.append(logging.StreamHandler(sys.stderr))
     if logconf['file']:
         handlers.append(logging.handlers.WatchedFileHandler(logconf['file']))
+    if (logging.root.hasHandlers()):
+        logging.root.handlers.clear()
```

Comment:
This doesn't seem to have anything to do with the timeouts. What is this doing?

```suggestion
    if logging.root.hasHandlers():
        logging.root.handlers.clear()
```

Reply:
Ah sorry, this was another change I wanted to submit. When logging to stdout, we see every entry twice. This fixes this problem. I will create another PR.